tests/app/conftest.py

```diff
@@ -629,13 +634,34 @@ def fake_uuid():
 
 
 @pytest.fixture(scope='function')
+def current_sms_provider():
+    return ProviderDetails.query.filter_by(
+        notification_type='sms'
+    ).order_by(
+        asc(ProviderDetails.priority)
+    ).first()
+
+
+@pytest.fixture(scope='function')
+def set_primary_sms_provider(identifier='mmg'):
```

Comment:
this isn't used as a fixture anywhere - just pop this function up the top of `test_provider_details_dao`